src/Engines/TestWare.Engines.Selenium/Factory/EdgeDriver.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium.Edge;
+
+namespace TestWare.Engines.Selenium.Factory;
+
+public class EdgeDriver : OpenQA.Selenium.Edge.EdgeDriver, IBrowserDriver
+{
+    public EdgeDriver()

```

Comment:
Default constructor doesn't need to be specified
https://www.geeksforgeeks.org/c-sharp-default-constructor/

Reply:
deleted